.github/workflows/check-r-versions.yml

```diff
@@ -69,3 +69,17 @@ jobs:
       r_versions: ${{ needs.check-r-versions.outputs.new_r_versions }}
       publish: ${{ inputs.publish || 'production' }}
     secrets: inherit
+
+  # Notify Hosted about new R versions
+  notify-slack-success:
+    needs: [build-new-r-versions, check-r-versions]
+    if: ${{ success() && needs.check-r-versions.outputs.new_r_versions != '' }}
```

Comment:
I'm not sure I'm following the logic here.  If what is successful?  Does that mean if both the `needs` are successful?

Reply:
Right, the intent was to only notify if both `needs` succeeded. Looking again at the GHA docs, it's unclear to me if this will really work though: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#status-check-functions

It talks about "previous steps" succeeding but I don't know if that covers dependent jobs in `needs`. I can keep looking here to see if there's a better documented way to do this.